docs/api/apiv3/components/schemas/file_links_model.yml

```diff
@@ -0,0 +1,113 @@
+# Schema: File_LinksModel
+---
+allOf:
+  - "$ref": './collection_model.yml'
+  - type: object
+    required:
+      - _links
+      - _embedded
+    properties:
+      _links:
+        type: object
+        required:
+          - self
+        properties:
+          self:
+            allOf:
+              - "$ref": "./link.yml"
+              - description: |-
+                  This file links list
+                  
+                  **Resource**: File_LinksModel
+                readOnly: true
+      _embedded:
+        type: object
+        required:
+          - elements
+        properties:
+          elements:
+            type: array
+            items:
+              $ref: './file_link_model.yml'
+
+example:
+  _type: Collection
+  total: 2
+  count: 2
+  _links:
+    self:
+      href: '/api/v3/work_packages/42/file_links'
+  _embedded:
+    elements:
+      - id: 1337
+        _type: FileLink
+        createdAt: '2021-12-20T13:37:00.211Z'
+        updatedAt: '2021-12-20T13:37:00.211Z'
+        originData:
+          id: 5503
+          name: logo.png
+          mimeType: image/png
+          createdAt: '2021-12-19T09:42:10.170Z'
+          lastModifiedAt: '2021-12-20T14:00:13.987Z'
+          createdByName: Luke Skywalker
+          lastModifiedByName: Anakin Skywalker
```

Comment:
We tried to keep with the `originData` closer to nextcloud, then to OpenProject. Maybe this was a mistake. But maybe this is not so big an issue. Especially, because these values are completely created by the client, while those fields on our side are created from metadata (author is currently logged in user, etc.)